modules/articles/client/services/articles.client.service.js

```diff
@@ -1,8 +1,13 @@
-'use strict';
+(function () {
+  'use strict';
 
-//Articles service used for communicating with the articles REST endpoints
-angular.module('articles').factory('Articles', ['$resource',
-  function ($resource) {
+  angular
+    .module('articles')
+    .factory('Articles', Articles);
+
+  Articles.$inject = ['$resource'];
+
+  function Articles($resource) {
```

Comment:
@trainerbill To follow John Papa to the T, this should be ArticlesService

https://github.com/johnpapa/angular-styleguide#naming